handlers/response/writer_test.go

```diff
@@ -0,0 +1,75 @@
+package response
```

Comment:
no reason not to do both - we can have multiple test files, one which tests the package internally and another which tests the external interface, if that adds value